From f1ea8523dca27265030904ea4ecf361453c2850c Mon Sep 17 00:00:00 2001
From: MengLi <meng.li@windriver.com>
Date: Fri, 5 Feb 2021 09:55:13 +0800
Subject: [PATCH 01/13] arch: arm64: s32g: disable virtio_block dts node by
 default

Get this patch from branch v5.10/standard/nxp-sdk-5.4/nxp-s32g2xx

There is no corresponding virtio device resource on SoC
S32g274a. So, disable the dts node.

Upstream-Status: Pending

Signed-off-by: Meng Li <Meng.Li@windriver.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-s32g.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-s32g.dtsi b/arch/arm64/boot/dts/freescale/fsl-s32g.dtsi
index 3823aa7ad890..2bbdb2eed652 100644
--- a/arch/arm64/boot/dts/freescale/fsl-s32g.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-s32g.dtsi
@@ -26,6 +26,7 @@ virtio_block@39501000 {
 		compatible = "virtio,mmio";
 		reg = <0x0 0x39501000 0x0 0x1000>;
 		interrupts = <0 68 4>;
+		status = "disabled";
 	};
 
 	reserved-memory {
-- 
2.25.1

